compiler/passes/normalize.cpp

```diff
@@ -1218,6 +1222,21 @@ static void normalizeReturns(FnSymbol* fn) {
   }
 }
 
+static void fixupExportedArrayReturns(FnSymbol* fn) {
+  if (fn->hasFlag(FLAG_EXPORT) && returnsArray(fn)) {
```

Comment:
How do you know that this will run only after makeExportWrapper runs?
It'd be bad to do this to the Chapel version before making the export wrapper, right?
Since Chapel code wouldn't be able to use it anymore?

So I'd expect this to check for FLAG_COMPILER_GENERATED and FLAG_EXPORT both, and at least a comment indicating why you know it runs after makeExportWrapper (since at least this relationship is not obvious to me).

Reply:
Right, good point.  I'd forgotten all the checks I put in place, I'll add that